tests/lib/libraries.js

```diff
@@ -34,9 +35,9 @@ describe("Libraries", () => {
 
             // var filename = "angular-1.2.5.js";
 
-            it("should produce correct AST when parsed", () => {
-                const output = shelljs.cat(`${filename}.result.json`);
-                const input = shelljs.cat(filename);
```

Comment:
This unit test was failing with a timeout on my macOS dev environment, probably because of the large size of the file tests/fixtures/libraries/angular-1.2.5.js.result.json. After switching to native Node.js API calls, all tests are passing again.